src/ancestry_proof.rs

```diff
@@ -321,3 +288,41 @@ fn calculate_root<
     let peaks_hashes = calculate_peaks_hashes::<_, M, _>(nodes, mmr_size, proof_iter)?;
     bagging_peaks_hashes::<_, M>(peaks_hashes)
 }
+
+pub fn expected_ancestry_proof_size(prev_mmr_size: u64, mmr_size: u64) -> usize {
```

Comment:
Will this be used outside of tests?

Do you plan to validate proofs in upper layers based on their expected size?

Reply:
Yes, I plan to call this from the `report_fork_voting()` extrinsic exposed in the beefy pallet, in order to check if the provided proofs are optimal.